design/identity_clusters.go

```diff
@@ -0,0 +1,39 @@
+package design
+
+import (
+	d "github.com/goadesign/goa/design"
+	a "github.com/goadesign/goa/design/apidsl"
+)
+
+var _ = a.Resource("identityClusters", func() {
```

Comment:
I'm not sure we should have a new base path such as `identityclusters`. What about using the actual `/api/clusters` base path and provide something like `POST api/clusters/:clusterID/identities` instead ?

and in the future, we could provide an endpoint to list all clusters for a user with `GET /api/clusters?identity=:identityID`

WDYT?

Reply:
@xcoulon: we don't have `clusterID` available while populating this relationship from auth. We call this endpoint while user registration where we have access to identityID and clusterURL. So we need to call cluster service to get clusterID and then do an post call.

Reply:
We could get clusterID (it already exists in the cluster service) but it would require additional call to Cluster Service or cluster cache refresh if it's missing in the cache.

@dipak-pawar @xcoulon I've updated the cluster management doc. Please take a look - https://docs.google.com/document/d/1_SndYYTnUV3hPMh26HXtUcrDAqBFJ4IrDqZA4b7jEtE/edit?usp=sharing